applications/GeoMechanicsApplication/tests/test_conditions/documentation_data/load_flux_domain.svg

```diff
@@ -0,0 +1 @@
-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 1.22465e-16 1.22465e-16 1 656 499)"/><path d="M676.5 574C676.5 579.247 672.023 583.5 666.5 583.5 660.977 583.5 656.5 579.247 656.5 574 656.5 568.753 660.977 564.5 666.5 564.5 672.023 564.5 676.5 568.753 676.5 574Z" stroke="#172C51" stroke-width="1.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd"/><path d="M676.5 522C676.5 527.247 672.023 531.5 666.5 531.5 660.977 531.5 656.5 527.247 656.5 522 656.5 516.753 660.977 512.5 666.5 512.5 672.023 512.5 676.5 516.753 676.5 522Z" stroke="#172C51" stroke-width="1.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd"/><path d="M0 0 28.8 28.8" stroke="#000000" stroke-width="2" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 316.8 666)"/><path d="M0 0 28.8 28.8" stroke="#000000" stroke-width="2" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 604.8 666)"/><path d="M0 0 28.8 28.8" stroke="#000000" stroke-width="2" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 556.8 666)"/><path d="M0 0 28.8 28.8" stroke="#000000" stroke-width="2" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 508.8 666)"/><path d="M0 0 28.8 28.8" stroke="#000000" stroke-width="2" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 460.8 666)"/><path d="M0 0 28.8 28.8" stroke="#000000" stroke-width="2" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 412.8 666)"/><path d="M0 0 28.8 28.8" stroke="#000000" stroke-width="2" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 364.8 666)"/><path d="M323.833 19.5001 323.833 70.7001 321.167 70.7001 321.167 19.5001ZM329.167 69.3667 322.5 82.7001 315.833 69.3667Z" fill="#4472C4"/><path d="M371.833 19.5001 371.833 70.7001 369.167 70.7001 369.167 19.5001ZM377.167 69.3667 370.5 82.7001 363.833 69.3667Z" fill="#4472C4"/><path d="M419.833 19.5001 419.833 70.7001 417.167 70.7001 417.167 19.5001ZM425.167 69.3667 418.5 82.7001 411.833 69.3667Z" fill="#4472C4"/><path d="M467.833 19.5001 467.833 70.7001 465.167 70.7001 465.167 19.5001ZM473.167 69.3667 466.5 82.7001 459.833 69.3667Z" fill="#4472C4"/><path d="M515.833 19.5001 515.833 70.7001 513.167 70.7001 513.167 19.5001ZM521.167 69.3667 514.5 82.7001 507.833 69.3667Z" fill="#4472C4"/><path d="M563.833 19.5001 563.833 70.7001 561.167 70.7001 561.167 19.5001ZM569.167 69.3667 562.5 82.7001 555.833 69.3667Z" fill="#4472C4"/><path d="M611.833 19.5001 611.833 70.7001 609.167 70.7001 609.167 19.5001ZM617.167 69.3667 610.5 82.7001 603.833 69.3667Z" fill="#4472C4"/><text font-family="Calibri,Calibri_MSFontService,sans-serif" font-style="italic" font-weight="700" font-size="24" transform="translate(261.847 46)">q</text><text font-family="Calibri,Calibri_MSFontService,sans-serif" font-weight="400" font-size="24" transform="translate(280.013 46)">[N]</text></g></svg>
```

Comment:
Nice start of a picture, some remarks:
- q loads or distributed line loads are usually schematized with a connecting line over the starts of the arrows. See the copyable thing in our team channel Documents/Schematics.svg ( only needs to be rotated )
- The support indication with skew lines usually connects to the line for which they apply and stay within the width of the line. See  the copyable thing in our team channel Documents/Schematics.svg